Library/Formula/casperjs.rb

```diff
@@ -1,22 +1,39 @@
-require 'formula'
-
 class Casperjs < Formula
-  homepage 'http://www.casperjs.org/'
-  url 'https://github.com/n1k0/casperjs/archive/1.0.4.tar.gz'
-  sha1 '4e1bdbf2ce93506b6c0d193c90a6f9bcd58c6254'
+  homepage "http://www.casperjs.org/"
+  url "https://github.com/n1k0/casperjs/archive/1.0.4.tar.gz"
+  sha256 "d71b9dd77ac202f3fbb958f8876f12b89aee2a1b09b2c2c55fd11aa928a1fb1f"
 
   devel do
-    url 'https://github.com/n1k0/casperjs/archive/1.1-beta3.tar.gz'
-    sha1 'd1c31f6a19c0b636f3e4dc17ffeb8224c8af8aef'
-    version '1.1-beta3'
+    url "https://github.com/n1k0/casperjs/archive/1.1-beta3.tar.gz"
+    sha256 "bc286424fb52df6cf16cb9b8ef6534ee830bb8ab0b87d2625910d8c1824152dc"
+    version "1.1-beta3"
   end
 
-  head 'https://github.com/n1k0/casperjs.git'
+  head "https://github.com/n1k0/casperjs.git"
+
+  # For embedded Phantomjs
+  depends_on :macos => :snow_leopard
 
-  depends_on 'phantomjs'
+  # https://github.com/Homebrew/homebrew/pull/38632
+  resource "phantomjs" do
+    url "https://phantomjs.googlecode.com/files/phantomjs-1.8.2-macosx.zip"
```

Comment:
Yes. CasperJS disagrees with you on supporting 1.9.8. Install 1.9.8, run CasperJS, and:

` CasperJS 1.0.x does not support PhantomJS version >= 1.9 `